.github/workflows/go-getter.yml

```
@@ -19,9 +19,9 @@ jobs:
     strategy:
       matrix:
         go-version:
-          - 1.17
-          - 1.18
           - 1.19
+          - "1.20"
```

Should this be a string?
```suggestion
          - 1.20
```

---

Yes as otherwise YAML "simplifies" it as `1.2` (float automatic casting), so this one has to be quoted.
The others don't get converted so they are fine without quoting.

---

We talked about this in slack but just for Git context I think you should add a comment here explaining that, gotta love YAML though 🤣